olive/evaluator/metric_config.py

```diff
@@ -100,6 +102,21 @@ def check_type(cls, v):
     def check_value(cls, v, values):
         if "type" not in values:
             raise ValueError("Invalid type")
-        if values["type"] in ["min-improvement", "max-degradation"] and v < 0:
-            raise ValueError(f"Value must be positive for type {values['type']}")
+        if (
+            values["type"]
+            in ["min-improvement", "max-degradation", "percent-min-improvement", "percent-max-degradation"]
```

Comment:
```suggestion
            in {"min-improvement", "max-degradation", "percent-min-improvement", "percent-max-degradation"}
```

nit: just flying by: use a set for membership checks

Reply:
https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/use-set-for-membership.html

Reply:
Updated.